src/command/KeyBindingManager.js

```diff
@@ -410,6 +411,13 @@ define(function (require, exports, module) {
             });
         }
         
+        // Also make sure that the second word of PageUp/PageDown has the first letter in upper case.
+        if (/^Page/.test(key)) {
+            key = key.replace(/(up|down)$/, function (match, p1) {
+                return p1.charAt(0).toUpperCase() + p1.substr(1);
+            });
+        }
```

Comment:
I tried creating shortcuts with `PAGEUP` and `PAGEDOWN` and they fail. Seems weird that case doesn't need to be exact, but only some case are accepted. Is that the intention?